pkg/commands/resolver.go

```diff
@@ -204,6 +223,7 @@ type nopPublisher struct {
 }
 
 func (n nopPublisher) Publish(_ context.Context, br build.Result, s string) (name.Reference, error) {
+	s = strings.TrimPrefix(s, build.StrictScheme)
```

Comment:
Why is this change necessary?

Reply:
Ah, that was unrelated - the nopPublisher failed in a unit test because the scheme prefix wasn't trimmed.